doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1251,46 +1259,50 @@ Response code:
 
 + Response 200 (application/json)
 
-        {
-          "Car": {
+        [
+          {
+            "type": "Car",
             "attrs": {
               "speed": {
-                "type": "none"
-              },
-              "fuel": {
-                "type": "gasoline"
+                "types": [ "none" ]
               },
               "fuel": {
-                "type": "diesel"
+                "types": [ "gasoline", "diesel" ]
```

Comment:
Not really... types in this array appear in the same orther the are retrieved from DB. Ordering them would have a computational cost and don't have too much value...